src/main/content/_includes/head.html

```diff
@@ -1,11 +1,19 @@
-<head>
+{% assign description = '' %}
+  {% if page.seo-description %}    
+    {% assign description = page.seo-description | escape %}
```

Comment:
I'm going a bit back and forth with myself whether we need to `escape` our descriptions.  On one hand, `escape` will make our descriptions less human readable.  On the other hand, `escape` is good security practice for special characters.  @steven1046 your thoughts?

I also, oddly, could not actually get `escape` to work.  When I inspect the description, the special characters are not "encoded".  I used my local test environment and added characters `??` to `:page-seo-description:` to one of the guides.  I inspected the HTML and still see the non-escaped `??` in the `description` 

![image](https://user-images.githubusercontent.com/31117513/48080089-81a06180-e1b2-11e8-8ab3-7c4e301f565c.png)